### PR TITLE
457- Add attention message

### PR DESCRIPTION
**Pull request by anaislet-c2c**
### JIRA issue

https://jira.camptocamp.com/browse/GSREN3D-457

### Description

Add attention message on "Quitter" and "Annuler" buttons. 

### Screenshots

image

### CODE REVIEW

**Review by toni-garcia-camptocamp — COMMENTED** [review]
LGTM, just some questions : 
- Maybe add a button "Annuler" ? Because here only button OK is display . "Annuler" can just close the alert 
- Is it possible to display this alert when the page is reload or the user click on the arrow to go back ?

File: src/components/simulation/FooterButtons.vue [hunk 1]
```diff
@@ -11,6 +11,9 @@ const router = useRouter()
 const simulationStore = useSimulationStore()
 
 function clickButtonCancel() {
```
Comment (ismailsunni) [inline src/components/simulation/FooterButtons.vue:13]: The function seems the same as in `displayErrorMessage`. Is there any reason to not make it as one function?

**Review by ismailsunni — COMMENTED** [review]
Not sure about the general design, but what if the user wants to cancel it? Is there a way to cancel? Or is this only information to the user?

**Comment by anaislet-c2c** [comment]
@ismailsunni  @toni-garcia-camptocamp  All your comments have been taken into account on this new PR : https://github.com/sigrennesmetropole/cooperation_jn_solar/pull/144

**Pull request closed without merging**